modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v3_7_3/DDMFormInstanceReportUpgradeProcess.java

```diff
@@ -265,12 +265,12 @@ private JSONObject _processDDMFormValues(
 		DDMFormInstanceReportUpgradeProcess.class);
 
 	private final DDMFormDeserializer _ddmFormDeserializer;
-	private final DDMFormFieldTypeReportProcessorTracker
-		_ddmFormFieldTypeReportProcessorTracker =
-			new DDMFormFieldTypeReportProcessorTracker();
+	private final DDMFormFieldTypeReportProcessorRegistry
+			_ddmFormFieldTypeReportProcessorRegistry =
+			new DDMFormFieldTypeReportProcessorRegistry();
 	private final JSONFactory _jsonFactory;
 
-	private class DDMFormFieldTypeReportProcessorTracker {
+	private class DDMFormFieldTypeReportProcessorRegistry {
```

Comment:
@slnn Looking at the logic, this inner class is actually a "factory", not a "registry". Also, it's only used by itself once, so I don't think a inner class is needed. Just move the method up, make it private, and call it.

Reply:
Done and has forced push to this pull